developers/blobstream.md

```diff
@@ -14,89 +14,111 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
-reference [the Blobstream smart contract](https://github.com/celestiaorg/blobstream-contracts/blob/master/src/Blobstream.sol).
+The Blobstream X zero-knowledge proof not only verifies the consensus of
+Celestia validators, but it also merkelizes and hashes all the data roots
+in the block range from the previous update to the current update, making
+accessible all Celestia data roots (verifiable with a Merkle inclusion proof
+against the stored Merkle root) to rollups.
 
-The **orchestrator** is part of the validator setup and works as follows:
+Blobstream X is built and deployed with
+[Succinct’s unified proving stack](https://succinct.xyz/).
 
-- celestia-app creates an attestation on the state machine level that needs to
-  be signed
-- The orchestrator queries the attestation, signs it, then submits the signature
-  to the Blobstream P2P network
+![blobstream x draft diagram](/img/blobstream/blobstream_x_draft_diagram.png)
 
-The **relayer** submits the attestations' signatures from the Blobstream
-P2P network to the target EVM chain.
+## Integrate with Blobstream X
 
-:::tip NOTE
-If the contract is still not deployed, then it needs to be
-deployed before it is used by the relayer. See the
-[deployment documentation](../nodes/blobstream-deploy.md) for more details.
-:::
+The following docs go over how developers can integrate Blobstream X.
 
-## How Blobstream works
+You can [find the repository for Blobstream X](https://github.com/succinctlabs/blobstreamx)
+along with code for:
 
-Blobstream allows Celestia block header data roots to be relayed in one
-direction, from Celestia to an EVM chain. It does not support bridging
-assets such as fungible or non-fungible tokens directly, and cannot send
-messages from the EVM chain back to Celestia.
+- The Blobstream X smart contract - [`BlobstreamX.sol`](https://github.com/succinctlabs/blobstreamx/blob/main/contracts/src/BlobstreamX.sol)
+- [The Blobstream X circuits](https://alpha.succinct.xyz/celestia/blobstreamx)
+- [The Blobstream X contract Golang bindings](https://github.com/succinctlabs/blobstreamx/blob/main/bindings/BlobstreamX.go)
 
-It works by relying on a set of signers to attest to some event on Celestia:
-the Celestia validator set. The Blobstream contract keeps track of the
-Celestia validator set by updating its view of the validator set with
-`updateValidatorSet()`. More than 2/3 of the voting power of the current
-view of the validator set must sign off on new relayed events, submitted with
-`submitDataRootTupleRoot()`. Each event is a batch of `DataRootTuple`s, with
-each tuple representing a single
-[data root (i.e. block header)](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#header).
-Relayed tuples are in the same order as Celestia block headers.
+Canonical deployments of Blobstream X will be maintained on the
+following chains: Arbitrum One, Base and Ethereum Mainnet. Every 1
```

Comment:
I think on Ethereum Mainnet it will be every 4 hours?

Reply:
We can change that once we go for the Ethereum mainnet. So far, I think we're focusing on Arbitrum one and Base, which will have 1-hour batches, if I understand correctly. Once we reach mainnet, we can update the documentation to reflect the batches times we're targetting

Reply:
SGTM